cfme/fixtures/service_fixtures.py

```diff
@@ -32,12 +32,8 @@ def catalog():
     catalog = "cat_" + fauxfactory.gen_alphanumeric()
     cat = Catalog(name=catalog,
                   description="my catalog")
-    try:
-        cat.create()
-        yield cat
-    finally:
-        if cat.exists:
-            cat.delete()
+    cat.create()
```

Comment:
Yes , I need  the catalog in Testing the links in SSUI .If the catalog is deleted the catalog item is no more visible in SSUI